engine/provider_aws.ml

```diff
@@ -229,13 +229,14 @@ module Aws : Provider_template.Provider = struct
       process_response resp
     in
     let%lwt _resp, body =
-      match%lwt repeat_until_ok send_command 10 with
+      match%lwt repeat_until_ok send_command 60 with
       | Ok (r, b) ->
           Lwt.return (r, b)
-      | Error _ ->
-          failwith
+      | Error (`Msg message, note) ->
+          Lwt.fail_with (Fmt.str
```

Comment:
Should we leave this as `failwith` (or even `Fmt.failwith`)? The lwt docs mention that it might be better to prefer `failwith` over `Lwt.fail_with` if we don't attempt to store or catch the rejected promise further up the chain since `failwith` will result in the runtime recording the backtrace.

Reply:
Fair question. In theory using the Lwt ppx should give decent backtraces. But that's not happening here so something is missing somewhere.